DeviceAdministration/Web/Scripts/Views/Device/DeviceIndex.js

```diff
@@ -245,7 +245,7 @@
             },
             "columns": [
                 {
-                    "data": "DeviceProperties.HubEnabledState",
+                    "data": "deviceProperties.hubEnabledState",
```

Comment:
Since we changed UI and the controllers at the same time, as long as they're not using them individually it shouldn't break anything.